packages/ecom-react-hooks/package.json

```
@@ -0,0 +1,54 @@
+{
+    "name": "ecom-react-hooks",
+    "version": "2.1.0-dev",
+    "description": "A library that provides hooks for fetching data from Commerce Cloud",
+    "author": "dev@mobify.com",
+    "license": "See license in LICENSE",
+    "engines": {
+      "node": "^14.0.0",
+      "npm": "^6.14.4 || ^7.0.0 || ^8.0.0"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/SalesforceCommerceCloud/pwa-kit.git"
+      },
+      "homepage": "https://github.com/SalesforceCommerceCloud/pwa-kit/tree/develop/packages/ecom-react-hooks#readme",
+      "bugs": {
+        "url": "https://github.com/SalesforceCommerceCloud/pwa-kit/issues"
+      },
+    "scripts": {
+      "build": "cross-env NODE_ENV=production internal-lib-build build && tsc --emitDeclarationOnly",
+      "build:watch": "watch 'npm run build' src/",
+      "lint": "npm run lint:js && tsc --noEmit",
+      "lint:fix": "npm run lint:js -- --fix",
+      "lint:js": "internal-lib-build lint \"**/*.{js,jsx,ts,tsx}\"",
+      "test": "internal-lib-build test",
+      "test:watch": "npm test -- --watch",
+      "test:inspect": "node --inspect-brk jest --runInBand",
+      "tsc:noEmit": "tsc --noEmit",
```

```suggestion
      "typecheck": "tsc --noEmit",
```
I'm not sure what the name should be. But I feel a better name is what describes the purpose of what `tsc --noEmit` does. It does not generate an output file but simply does type checking.

If you want inspiration for the name, here's one from commerce-sdk-isomorphic: https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/blob/main/package.json#L26

---

Furthermore, once we have this script, we can call it from other scripts. For example:

`"lint": "npm run lint:js && npm run typecheck",`

---

Thanks for the suggestion @vmarta . I'll go with typecheck as you suggested since I like how clear `npm run typecheck` is on what you are doing.